kubernetes/loculus/values.yaml

```diff
@@ -1234,13 +1234,23 @@ defaultOrganisms:
         version: 1
         configFile:
           <<: *preprocessingConfigFile
+          require_nextclade_sort_match: True
+          minimizer_path: https://raw.githubusercontent.com/nextstrain/nextclade_data/refs/heads/ebola/data_output/minimizer_index.json
```

Comment:
We should not use the githubusercontent links due to it getting rate limited, let's create a loculus resources repo and use GitHub pages to avoid rate limits?

also let's call it URL not path? as path is only part of a URL